include/seqan3/range/detail/inherited_iterator_base.hpp

```diff
@@ -107,6 +107,21 @@ class inherited_iterator_base : public std::conditional_t<std::is_pointer_v<base
     {}
     //!\}
 
+    //!\brief Get a copy of the base.
```

Comment:
@rrahn Maybe you have a better description in mind, because it does not completely fit for the `&&` overload